server/events.lua

```diff
@@ -205,25 +205,6 @@ RegisterNetEvent('QBCore:ToggleDuty', function()
     TriggerClientEvent('QBCore:Client:SetDuty', src, Player.PlayerData.job.onduty)
 end)
 
--- Non-Chat Command Calling (ex: qb-adminmenu)
-
-RegisterNetEvent('QBCore:CallCommand', function(command, args)
-    local src = source
-    if not QBCore.Commands.List[command] then return end
-    local Player = QBCore.Functions.GetPlayer(src)
-    if not Player then return end
-    local hasPerm = QBCore.Functions.HasPermission(src, "command."..QBCore.Commands.List[command].name)
-    if hasPerm then
-        if QBCore.Commands.List[command].argsrequired and #QBCore.Commands.List[command].arguments ~= 0 and not args[#QBCore.Commands.List[command].arguments] then
-            TriggerClientEvent('QBCore:Notify', src, Lang:t('error.missing_args2'), 'error')
-        else
-            QBCore.Commands.List[command].callback(src, args)
-        end
-    else
-        TriggerClientEvent('QBCore:Notify', src, Lang:t('error.no_access'), 'error')
-    end
-end)
```

Comment:
Does this need to be deprecated or is it not a breaking change to remove this event?

Reply:
huuummm probably needs to be deprecated i'll figure something out tommorow